opentracing/src/main/java/io/opentracing/Tracer.java

```diff
@@ -87,15 +87,18 @@
       /** Same as {@link Span#setTag(String, String)}, but for the span being built. */
       SpanBuilder withTag(String key, String value);
 
-      /** Same as {@link Span#setTag(String, String)}, but for the span being built. */
+      /** Same as {@link Span#setTag(String, boolean)}, but for the span being built. */
       SpanBuilder withTag(String key, boolean value);
 
-      /** Same as {@link Span#setTag(String, String)}, but for the span being built. */
+      /** Same as {@link Span#setTag(String, Number)}, but for the span being built. */
       SpanBuilder withTag(String key, Number value);
 
       /** Specify a timestamp of when the Span was started, represented in microseconds since epoch. */
       SpanBuilder withStartTimestamp(long microseconds);
 
+      /** Same as {@link Span#setBaggageItem(String, String)}, but for the span being built. */
+      SpanBuilder withBaggage(String key, String value);
```

Comment:
@toaler, I just wrote a more detailed document about cross-process propagation (i.e., Inject and Join) here: http://opentracing.io/propagation/

I think that should hopefully be an incremental improvement in terms of understanding, although there is still more work to be done there (in particular, there ought to be more detail about *implementing* Inject and Join).